### ISSUE TITLE
[feature request] attribut on parameter to mute while "play from here"

### ISSUE THREAD

**Issue by pach** [ossia/score#651]
In some (rare) case, it can be usefull to not send OSC value on some specific parameter while doing a "play from here"

**Comment by jcelerier** [comment]
This should certainly be a "custom" attribute only available in score, that would be implemented like the ones in libossia's node_attributes.hpp.
Then, when doing play from here, before sending the state (`DefaultClockManager::prepareExecution`, `offset_state`) its messages should be filtered according to this

**Comment by bltzr** [comment]
I agree with this, but in my experience, I always needed this for parameters like a "play" trigger for a sound file or such - was that the same for you, @pach  ?
if so, in my usage, at least, that's always parameters ```@access set``` - is it the same for you @pach ? also @jln- ?
if so, maybe we could only include parameters ```@access bi``` in the play from here ?
